src/applications/claims-status/containers/AppealInfo.jsx

```diff
@@ -103,14 +103,11 @@ export class AppealInfo extends React.Component {
       appealContent = (
         <div>
           <div>
-            <Breadcrumbs>
-              <li>
-                <Link to="your-claims">Track Your Claims and Appeals</Link>
-              </li>
-              <li>
-                <strong>{claimHeading}</strong>
-              </li>
-            </Breadcrumbs>
+            <ClaimsBreadcrumbs>
+              <Link to={`/appeals/${appeal.id}`} key="claims-appeal">
```

Comment:
I think this should be a relative url (`appeals/${appeal.id}` instead of `/appeals/${appeal.id}`)